bluesky/protocols.py

```diff
@@ -151,26 +171,20 @@ def collect(self) -> Generator[Dict[str, Any], None, None]:
         """
```

Comment:
I guess collect() output is https://blueskyproject.io/event-model/data-model.html#event-document but without `seq_num`, `uid` and `descriptor`?